### ISSUE TITLE
Document how to include node properties in CSV data

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#57]
Like status, owner, etc.

**Comment by mjordan** [comment]
POSTing the following JSON works for 'status' but not 'sticky' or 'promote':

```
                    'status': [
                        {'value': False }
                    ],
                    'promote': [
                        {'value': False }
                    ],
                    'sticky': [
                        {'value': False }
                    ],    
```
Maybe we can go with status now and figure out the others when the need arises.

**Comment by mjordan** [comment]
And maybe at the batch level for now, not at the node level, e.g. it would be an option in the config file, not a value in the CSV file.

**Comment by mjordan** [comment]
Closed with 29e7803e807829967aee05782a209739ec749fb6.
